examples/trossen_ai/main.py

```diff
@@ -24,16 +24,20 @@
 import numpy as np
 from openpi_client import websocket_client_policy
 import cv2
-from lerobot.robots import make_robot_from_config
-from lerobot.robots.bi_widowxai_follower.config_bi_widowxai_follower import BiWidowXAIFollowerConfig
-from lerobot.cameras.realsense.configuration_realsense import RealSenseCameraConfig
 from collections import defaultdict
 from scipy.interpolate import PchipInterpolator
 
+from lerobot.cameras.realsense.configuration_realsense import RealSenseCameraConfig
+from lerobot.robots import (  # noqa: F401
+    make_robot_from_config,
+)
+from lerobot_robot_trossen.config_bi_widowxai_follower import BiWidowXAIFollowerRobotConfig
```

Comment:
lerobot and lerobot_trossen* are external imports and should be grouped with other external imports